### ISSUE TITLE
Step fails to create SDK Manager

### ISSUE THREAD

**Issue by danut-iosif** [bitrise-steplib/steps-install-missing-android-tools#9]
Hello!

I've just updated to 1.0.0 (since I was affected by the now-closed issue since I'm on a Cordova project) and executed the step as part of an Android build workflow.

The step now fails with the following:
Ensure dependencies
Failed to create sdk manager, error: android home not exists at: /usr/local/opt/android-sdk

Problem is, $ANDROID_HOME is set to that folder in my .zshrc and the SDK is actually, properly installed there, as every other tool that requires it can detect it.

What other information should I provide so that this can be better looked at?

Thank you!

**Comment by viktorbenei** [comment]
Hi @danut-iosif 

Thanks for reporting the issue here! ;)

@godrei / @trapacska you are summoned! ;)

**Comment by danut-iosif** [comment]
```
+------------------------------------------------------------------------------+
| (1) And some more needed Android stuffs...                                   |
+------------------------------------------------------------------------------+
| id: install-missing-android-tools                                            |
| version: 1.0.0                                                               |
| collection: https://github.com/bitrise-io/bitrise-steplib.git                |
| toolkit: go                                                                  |
| time: 2017-03-23T17:29:41Z                                                   |
+------------------------------------------------------------------------------+
|                                                                              |
INFO[17:29:41]  * [OK] Step dependency (go) installed, available.

Configs:
- RootBuildGradleFile: ./platforms/android/build.gradle
- GradlewPath: ./platforms/android/gradlew
- UpdateSupportLibraryAndPlayServices: true
- AndroidHome: /usr/local/opt/android-sdk

Search for root setting.gradle file
root build.gradle file: /Users/diosif/DanProjects/fourth_engage_bitrise/salesforce-hybrid/platforms/android/build.gradle
root settings.gradle file:: /Users/diosif/DanProjects/fourth_engage_bitrise/salesforce-hybrid/platforms/android/settings.gradle

Collect build.gradle files to analyze
active modules to analyze: []
build.gradle files to analyze: [/Users/diosif/DanProjects/fourth_engage_bitrise/salesforce-hybrid/platforms/android/build.gradle]

Collect dependencies to ensure
Analyze build.gradle file: /Users/diosif/DanProjects/fourth_engage_bitrise/salesforce-hybrid/platforms/android/build.gradle
failed to parse build gradle file: /Users/diosif/DanProjects/fourth_engage_bitrise/salesforce-hybrid/platforms/android/build.gradle, error: failed to determine compileSDKVersion, error: failed to find compileSdkVersion
switch to extended analyzer...
Failed to analyze build.gradle at: /Users/diosif/DanProjects/fourth_engage_bitrise/salesforce-hybrid/platforms/android/build.gradle, error: failed to parse compileSDKVersion ()

Ensure dependencies
Failed to create sdk manager, error: android home not exists at: /usr/local/opt/android-sdk
|                                                                              |
+---+---------------------------------------------------------------+----------+
| x | And some more needed Android stuffs... (exit code: 1)         | 0.71 sec |
+---+---------------------------------------------------------------+----------+
| Issue tracker: ...bitrise-steplib/steps-install-missing-android-tools/issues |
| Source: ...://github.com/bitrise-steplib/steps-install-missing-android-tools |
+---+---------------------------------------------------------------+----------+
```

**Comment by viktorbenei** [comment]
Thanks for the logs @danut-iosif !

Can you please run these in your Terminal and copy paste the outputs here:

```
echo "$ANDROID_HOME"
ls -alh /usr/local/opt/android-sdk
tree -L 2 /usr/local/opt/android-sdk
```

(If you don't have `tree` installed you can install it with `brew install tree`)

**Comment by danut-iosif** [comment]
```
~
❯ echo "$ANDROID_HOME"
/usr/local/opt/android-sdk

~
❯ ls -alh /usr/local/opt/android-sdk
lrwxr-xr-x  1 diosif  admin    30B  4 Oct 09:30 /usr/local/opt/android-sdk -> ../Cellar/android-sdk/24.4.1_1

~
❯ tree -L 2 /usr/local/opt/android-sdk
/usr/local/opt/android-sdk
├── INSTALL_RECEIPT.json
├── README
├── add-ons -> ../../../var/lib/android-sdk/add-ons
├── bin
│   ├── aapt
│   ├── adb
│   ├── aidl
│   ├── android
│   ├── ddms
│   ├── dexdump
│   ├── dmtracedump
│   ├── draw9patch
│   ├── dx
│   ├── emulator
│   ├── emulator-arm
│   ├── emulator-x86
│   ├── etc1tool
│   ├── fastboot
│   ├── hierarchyviewer
│   ├── hprof-conv
│   ├── lint
│   ├── llvm-rs-cc
│   ├── mksdcard
│   ├── monitor
│   ├── monkeyrunner
│   ├── traceview
│   └── zipalign
├── build-tools
│   ├── 23.0.1
│   ├── 23.0.2
│   └── 23.0.3
├── etc
│   └── bash_completion.d
├── extras -> ../../../var/lib/android-sdk/extras
├── platform-tools
│   ├── NOTICE.txt
│   ├── adb
│   ├── api
│   ├── dmtracedump
│   ├── etc1tool
│   ├── fastboot
│   ├── hprof-conv
│   ├── lib
│   ├── source.properties
│   ├── sqlite3
│   └── systrace
├── platforms -> ../../../var/lib/android-sdk/platforms
├── samples -> ../../../var/lib/android-sdk/samples
├── sources -> ../../../var/lib/android-sdk/sources
├── system-images -> ../../../var/lib/android-sdk/system-images
├── temp -> ../../../var/lib/android-sdk/temp
└── tools
    ├── NOTICE.txt
    ├── android
    ├── ant
    ├── apps
    ├── bin
    ├── bin64
    ├── ddms
    ├── draw9patch
    ├── emulator
    ├── emulator-check
    ├── emulator64-arm
    ├── emulator64-crash-service
    ├── emulator64-mips
    ├── emulator64-x86
    ├── hierarchyviewer
    ├── jobb
    ├── lib
    ├── lib64
    ├── lint
    ├── mksdcard
    ├── monitor
    ├── monkeyrunner
    ├── proguard
    ├── qemu
    ├── screenshot2
    ├── source.properties
    ├── support
    ├── templates
    ├── traceview
    └── uiautomatorviewer

29 directories, 53 files
```

**Comment by viktorbenei** [comment]
Thanks @danut-iosif !

**Comment by danut-iosif** [comment]
Hah, always a pleasure, mate! :)

**Comment by trapacska** [comment]
Hey @danut-iosif ! 👋🏻

Thank you for reporting the issue! With the new release it is resolved now! (#10) 
Don't forget to update the step in your workflow! 😉
🚀🎉
